plugins/modules/cloud/rackspace/rax_mon_check.py

```diff
@@ -20,61 +20,71 @@
   rax_mon_alarm
 options:
   state:
+    type: str
     description:
     - Ensure that a check with this C(label) exists or does not exist.
     choices: ["present", "absent"]
+    default: present
   entity_id:
+    type: str
     description:
     - ID of the rax_mon_entity to target with this check.
     required: true
   label:
+    type: str
     description:
     - Defines a label for this check, between 1 and 64 characters long.
     required: true
   check_type:
+    type: str
     description:
     - The type of check to create. C(remote.) checks may be created on any
       rax_mon_entity. C(agent.) checks may only be created on rax_mon_entities
       that have a non-null C(agent_id).
-    choices:
-    - remote.dns
-    - remote.ftp-banner
-    - remote.http
-    - remote.imap-banner
-    - remote.mssql-banner
-    - remote.mysql-banner
-    - remote.ping
-    - remote.pop3-banner
-    - remote.postgresql-banner
-    - remote.smtp-banner
-    - remote.smtp
-    - remote.ssh
-    - remote.tcp
-    - remote.telnet-banner
-    - agent.filesystem
-    - agent.memory
-    - agent.load_average
-    - agent.cpu
-    - agent.disk
-    - agent.network
-    - agent.plugin
+    - |
+        Choices for this option are:
+        - remote.dns
```

Comment:
```suggestion
        - C(remote.dns)
```
(same for the next ones)